source/AsyncRgbLedAnalyzerSettings.cpp

```diff
@@ -1,63 +1,180 @@
 #include "AsyncRgbLedAnalyzerSettings.h"
 #include <AnalyzerHelpers.h>
 
+const char* DEFAULT_CHANNEL_NAME = "Addressable LEDs (Async)";
+
+double operator "" _ns(unsigned long long x)
```

Comment:
lets use U64 here instead of unsigned long long for the type. Eventually I would live to convert the entire Logic project over to cstdint types, but for now let's use the saleae ones for consistency.

Reply:
Unfortunately we can't - I tried - the C++11 spec for postfix operators is very picky about the types (I don't know the reasoning), this type has to be explicitly 'unsigned long long'.